handbook/README.md

```diff
@@ -1,10 +1,18 @@
 # Club Handbook
 
-Welcome to the [UBC Launch Pad](https://ubclaunchpad.com/) handbook! Launch Pad-wide documentation is hosted in these sections - if you're not sure about something, this is the first place to check.
+Welcome to the [UBC Launch Pad](https://ubclaunchpad.com/) handbook! Launch Pad-wide documentation is hosted in these sections - if you're not sure about something, this is the first place to check. The easiest way to find something is to run a search in the navigation bar!
+
+Our goals with this handbook are to:
+
+* 📝 Create a comprehensive set of documentation that anyone can leverage to participate in and run Launch Pad - with written processes available, everyone has an equal opportunity to understand what we do and participate, rather than just those "in the know".
+* 🧠 Accumulate resources and lessons learned so that future generations of Launch Pad can focus on iterating and improving, rather than figuring out how to run the club from scratch every year.
+* 💪 Empower members find information they need independently, and hopefully enable others to learn from our work.
```

Comment:
"Empower members **to** ..."

Reply:
ahh, good catch!

Also, heads up that you can make [direct suggestions on PRs that I can apply](https://docs.ubclaunchpad.com/handbook/tools/github#leaving-comments) 😄

Reply:
Woaaaah, I'm already learning from those docs. Damn I actually thought I was at the official github docs for a second.